libraries/browser-tracker-core/src/tracker/index.ts

```diff
@@ -642,9 +642,14 @@ export function Tracker(
     /*
      * Load the domain user ID and the session ID
      * Set the cookies (if cookies are enabled)
+     * If anonymous tracking is enabled, it clears the anonymised identifiers
```

Comment:
Nit:
`If anonymous tracking is enabled, it clears the anonymised identifiers`
to
`If anonymous tracking is enabled, it clears the stored identifiers`

Seems more clear to me on what it does practically. WDYT ?

Reply:
Sounds better, changed it!